src/annotator/sidebar.js

```diff
@@ -449,4 +450,22 @@ export default class Sidebar extends Guest {
   setAllVisibleHighlights(shouldShowHighlights) {
     this.crossframe.call('setVisibleHighlights', shouldShowHighlights);
   }
+
+  /**
+   * Shows the sidebar's controls
+   */
+  showControls() {
+    if (this.frame) {
+      this.frame.classList.remove('controls-hide');
```

Comment:
I think this className could use some adjustment to better match our conventions. This is a state-related class name, and we usually use the form `.is-<something>`

 The presence of this class causes the entire annotator frame to be hidden, not just the controls. So even though the net effect is that controls get hidden, it's really the entire frame, right?

As such, perhaps `.is-hidden` would be more appropriate?

Reply:
No, it only hides the controls.

`.is-controls-hidden` a better className?

Reply:
> No, it only hides the controls.

So, if the sidebar is open/visible, it would _only_ hide the controls?

Reply:
That was the original intention... but actually works hiding the _whole_ sidebar. This has the advantage that the it keeps the original state of the sidebar (either open or close).

I renamed the methods to `show` and `hide`.